apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -653,6 +654,14 @@ export class InstallManager {
         }
       }
 
+      if (!isEmpty(packageJson.resolutions)) {
```

Comment:
We should throw an exception if this is non-empty and we're not using Yarn, and we probably shouldn't populate this field for non-Yarn package managers.